psycopg/replication_cursor_type.c

```diff
@@ -231,6 +231,17 @@ psyco_repl_curs_get_io_timestamp(replicationCursorObject *self)
     return res;
 }
 
+/* object member list */
+
+#define OFFSETOF(x) offsetof(replicationCursorObject, x)
+
+static struct PyMemberDef replicationCursorObject_members[] = {
+    {"wal_end", T_ULONGLONG, OFFSETOF(wal_end), READONLY,
+        "LSN position of the current end of WAL on the server."},
+    {NULL}
```

Comment:
Would there be value in adding attributes to access the other LSN members too?

Reply:
I thought about adding them, but decided not to b/c I didn't need them and wanted to keep the PR as small as possible.

Reply:
Makes sense to make it in a different branch, yes.